django_cyverse_auth/protocol/globus.py

```diff
@@ -85,7 +85,7 @@ def globus_profile_for_token(globus_user_token):
 def _extract_expiry_date(epoch_secs):
     if type(epoch_secs) != int:
         try:
-            epoch_secs = int(date_int)
+            epoch_secs = int(epoch_secs)
```

Comment:
This is the only thing that makes sense to me.  They pass in the epoch_secs.  Since epoch_secs may be a float, they truncate it to seconds by casting it.  date_int isn't defined or used anywhere else.